<launch>
    <node name="visual_detector" pkg="udg_pandora" type="visual_detector.py" output="screen" />
    <!-- <node name="visual_detector_ee" pkg="udg_pandora" type="visual_detector_ee.py" output="screen"/> -->
    <node name="valve_tracker" pkg="udg_pandora" type="valve_tracker.py" />
    <node name="stare_landmark" pkg="cola2_control" type="stare_landmark.py" />
</launch>
